### ISSUE TITLE
"DX BUDDiES PLUS" support needed

### ISSUE THREAD

**Issue by Fun10165** [Neskol/MaiLib#50]
I was converting some charts from DX BUDDiES PLUS using MaichartConverter.
However, it keeps showing "System.IndexOutOfRangeException" and some random irrelevant errors.
I edited 22 to 21 in <AddVersion><id>22</id><str>...</str></AddVersion> in Music.xml, and it works.
And according to the source code, DX BUDDiES PLUS is not enumerated. I think the problem happened here.

**Comment by Neskol** [comment]
Yeah sorry I forgot to update this. Now it should work with PRiSM as well
